spec/openapi.yaml

```diff
@@ -480,6 +480,49 @@ paths:
         '500':
           description: An unexpected error occured.
 
+  /healthcheck/failure/notify:
+    post:
```

Comment:
as this is idempotent, should methods be PUT or simpler as all post ?  no firm view either way